### DISCUSSION TITLE
Limit String Length/ Reset String on change

### DISCUSSION THREAD

**Discussion by falkobehrens** [cables-gl/cables_docs#168] [Answers & Questions] [answered]
Hey dear cables community, 

i am trying to build something like a Type Tester but not for standard fonts but 3D files I modeled. 
The problem I have that I only want to show the last letter that was typed in... so I need to limit or reset the string input when it is changed. Do I need to build a custom opp for that?… or maybe you have a different approach.
Would love to hear from you.

Here is my patch: https://cables.gl/p/6210ce7209447773bd6c48fe
For the purpose of the proof of concept I just imported the ›a‹ and ›b‹

Best regards
Falko

**Comment by positlabs** [comment, marked as answer]
You can get the string length and then use substring to get the last character.

<img width="430" alt="Screen Shot 2022-02-20 at 2 03 52 PM" src="https://user-images.githubusercontent.com/1918844/154859685-c0bac384-2a19-48f1-ba40-46c7c1dd0a99.png">

**Reply by falkobehrens** [reply]
Genius :) Thanks a lot!
